Project/Source/Utils/ParticleMotionState.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "Components/Component.h"
+#include "Components/ComponentParticleSystem.h"
+#include "LinearMath/btMotionState.h"
+#include "LinearMath/btTransform.h"
+
+#include "Math/float3.h"
+
+class ParticleMotionState : public btMotionState {
+public:
+	ParticleMotionState(ComponentParticleSystem::Particle* p);
+	~ParticleMotionState();
+
+	void getWorldTransform(btTransform& centerOfMassWorldTrans) const;
+	void setWorldTransform(const btTransform& centerOfMassWorldTrans);
```

Comment:
Code Style:
Remember that functions and methods start with an uppercase.

Reply:
but this are overrides of btMotionState virtual methods.